src/Instrumentation/PDO/src/PDOTracker.php

```diff
@@ -106,4 +107,35 @@ private static function mapDriverNameToAttribute(?string $driverName): string
             default => 'other_sql',
         };
     }
+
+    /**
+     * Extracts attributes from a DSN string
+     *
+     * @param string $dsn
+     * @return array
+     */
+    private static function extractAttributesFromDSN(string $dsn): array
+    {
+        $attributes = [];
+        if (str_starts_with($dsn, 'sqlite')) {
+            $attributes[TraceAttributes::DB_NAME] = $dsn;
+
+            return $attributes;
+        }
+
+        if (preg_match('/user=([^;]*)/', $dsn, $matches)) {
```

Comment:
should some of these fields be left unset rather than 'unknown' ?

Reply:
yes! - done